roles/pulp3/tasks/configure.yml

```diff
@@ -1,28 +1,56 @@
 ---
 - block:
+    # Become root so as to search paths like /usr/sbin.
+    - name: Find the nologin executable
```

Comment:
Generally speaking, I had to move a chunk of install and configure around. This is not the prettiest solution currently. A container based setup needs some of the aspects a base install needs but not all. Mostly this is around configuration and data storage:

 * /etc/pulp
 * /var/lib/pulp

Those two need to be setup and configured prior to starting the containers.